### PR TITLE
Whatsapp - Send Text Using Template with support for dynamic URL in buttons

### LINKED ISSUE #5899: [ACTION] Send Whatsapp Text Using Template with support for dynamic URL in buttons
**Is there a specific app this action is for?**

Send Whatsapp Text Using Template with support for dynamic URL in buttons

**Please provide a link to the relevant API docs for the specific service / operation.**

https://developers.facebook.com/docs/whatsapp/api/messages/message-templates/interactive-message-templates

I currently have this error message when I am try to use the current "Send Text Using Template" action with a template that requires a dynamic url

`Error - Request failed with status code 400
{"error":{"message":"(#131008) Required parameter is missing","type":"OAuthException","code":131008,"error_data":{"messaging_product":"whatsapp","details":"buttons: Button at index 0 of type Url requires a parameter"},"fbtrace_id":"AEPaERTvMqsfcyJDi_VKjTO"}}`

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #5899 

Update to prevent button parameters from being added to body parameters.

### CODE REVIEW

File: components/whatsapp_business/actions/send-text-using-template/send-text-using-template.mjs [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default {
   key: "whatsapp_business-send-text-using-template",
   name: "Send Text Using Template",
   description: `Send a text message using a pre-defined template. Variables can be sent only as text. [See the docs.](${docLink})`,
-  version: "0.0.3",
+  version: "0.0.4",
```
Comment (dylburger) [inline components/whatsapp_business/actions/send-text-using-template/send-text-using-template.mjs:10]:
```suggestion
  version: "0.0.4",
```

Comment (dylburger) [reply components/whatsapp_business/actions/send-text-using-template/send-text-using-template.mjs:10]: Removed extra double quotes around 0.0.4

File: components/whatsapp_business/actions/send-text-using-template/send-text-using-template.mjs [hunk 2]
```diff
@@ -113,7 +113,7 @@ export default {
     }
 
     const bodyParameters = Object.keys(this)
-      .filter((key) => key.match(regex) && key !== "header_{{1}}")
+      .filter((key) => key.match(regex) && key !== "header_{{1}}" && key !== "button_{{1}}")
```
Comment (dylburger) [inline components/whatsapp_business/actions/send-text-using-template/send-text-using-template.mjs:116]:
```suggestion
      .filter((key) => key.match(regex) && key !== "header_{{1}}" && key !== "button_{{1}}")
```

Comment (dylburger) [reply components/whatsapp_business/actions/send-text-using-template/send-text-using-template.mjs:116]: No changes needed

File: components/whatsapp_business/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/whatsapp_business",
-  "version": "0.0.7",
+  "version": "0.0.8",
```
Comment (dylburger) [inline components/whatsapp_business/package.json:3]:
```suggestion
  "version": "0.0.8",
```

Comment (dylburger) [reply components/whatsapp_business/package.json:3]: No changes needed

**Review by feyzullah — APPROVED** [review]

**Pull request merged**